plugins/styled-bars/back.js

```diff
@@ -55,12 +55,9 @@ function switchMenuVisibility() {
 function fixMenu(template) {
 	for (let index in template) {
 		let item = template[index];
-		//apply function on submenu
-		if (item.submenu != null) {
-			fixMenu(item.submenu);
-		}
-		//change onClick of checkbox+radio
-		else if (item.type === 'checkbox' || item.type === 'radio') {
+		//change onClick of checkbox+radio if not fixed
+		if ((item.type === 'checkbox' || item.type === 'radio') && !item.fixed) {
+			item.fixed = true;
```

Comment:
useless if everything works well, but guarantee that original onClick won't be overwritten more than once